PROJECT

```diff
@@ -21,7 +21,6 @@ resources:
   path: github.com/openshift/dpu-operator/api/v1
```

Comment:
can you fix having to manually do `go get github.com/onsi/ginkgo/v2@v2.20.2`

Reply:
the upgrade is done to avoid some failure in `make test`. Is there a problem, or what's the use of staying with an old version (and a different version from the toplevel `go.mod`)?

Reply:
without `(cd api/ && go get github.com/onsi/ginkgo/v2@v2.20.2)`, a subsequent `make test` gives a warning:

```
$ make test
GOFLAGS='' /data/src/dpu-operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
for d in . dpu-api api tools ; do \
        if [ "$d" = . ] ; then \
                (cd $d && go mod vendor) || exit $? ; \
        fi ; \
        (cd $d && go mod tidy) || exit $? ; \
done
GOFLAGS='' /data/src/dpu-operator/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
test -s /data/src/dpu-operator/bin/setup-envtest || GOBIN=/data/src/dpu-operator/bin GOFLAGS='' go install sigs.k8s.io/controller-runtime/tools/setup-envtest@release-0.16
test -s /data/src/dpu-operator/bin/ginkgo && /data/src/dpu-operator/bin/ginkgo version | grep -q v2.20.2 || \
GOBIN=/data/src/dpu-operator/bin GOFLAGS='' go install github.com/onsi/ginkgo/v2/ginkgo@v2.20.2
FAST_TEST=false KUBEBUILDER_ASSETS="/data/src/dpu-operator/bin/k8s/1.27.1-linux-amd64" /data/src/dpu-operator/bin/ginkgo --repeat 4  ./... -coverprofile cover.out
Ginkgo detected a version mismatch between the Ginkgo CLI and the version of Ginkgo imported by your packages:
  Ginkgo CLI Version:
    2.20.2
  Mismatched package versions found:
    2.14.0 used by v1

  Ginkgo will continue to attempt to run but you may see errors (including flag
  parsing errors) and should either update your go.mod or your version of the
  Ginkgo CLI to match.

  To install the matching version of the CLI run
    go install github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file.  Alternatively you can use
    go run github.com/onsi/ginkgo/v2/ginkgo
  from a path that contains a go.mod file to invoke the matching version of the
  Ginkgo CLI.

  If you are attempting to test multiple packages that each have a different
  version of the Ginkgo library with a single Ginkgo CLI that is currently
  unsupported.
  
[1732290406] Webhook Suite - 3/3 specs ••• SUCCESS! 8.971230559s PASS
[1732290406] Cnihelper Suite - 1/1 specs • SUCCESS! 984.902µs PASS
...
```

Reply:
`make gingko`

Reply:
do you mean, to add a make target for calling `go get ginko`?

Reply:
I mean that we already have that target.

Reply:
I think that is not correct.

Reply:
what's wrong with that?

Reply:
there is no make target `make gingko` in dpu-operator `main` branch.

Reply:
https://github.com/openshift/dpu-operator/blob/main/Makefile#L342